src/Modules/Extensions/templates/Backend/Actions/ThemeIndex.html.twig

```diff
@@ -82,42 +82,42 @@
       </div>
       <div class="card-body">
         <div id="installableThemes" class="row row-cols-1 row-cols-sm-2 row-cols-md-3 row-cols-lg-2 row-cols-xl-3 g-4">
-          {% for installableThemeData in installableThemes %}
-            {% set installableTheme = installableThemeData.theme %}
-            {% set themeInstallForm = installableThemeData.installForm %}
+          {% for installable_theme_data in installableThemes %}
```

Comment:
and here installable_theme

Reply:
@bjorvack this has not been processed?

Reply:
fixed f113565f4902ae3899f25cbc56cac3c727ae933b